### PR TITLE
Add recently joined team members as code owners

### PR DESCRIPTION
**Pull request by rplevy**

### CODE REVIEW

**Review by ereteog — CHANGES_REQUESTED** [review]
Some parts are too sensitive, we can include sub directories when I am back.

**Review by agzam — APPROVED** [review]

**Review by ereteog — APPROVED** [review]
I am on PTO and cannot attend ongoing discussions.
Let's stick to my general rule: trust by default but don't mess with it 😉.
I would like to check ES related changes, mapping errors are VERY expensive to fix.

**Review by irinarenteria — APPROVED** [review]

**Review by wandersoncferreira — APPROVED** [review]

**Comment by rplevy** [comment]
Resolution on CTIA codeowners is that the codeowners file is not actually enforced (we have never had the "require code owner approval" box checked) but @gbuisson says let's leave the file anyway as it pertains to notifying people.

**Pull request merged**
